provider/pkg/gen/nodejs-templates/helm/v2/helm.ts

```diff
@@ -143,6 +143,7 @@ export class Chart extends yaml.CollectionComponentResource {
         config: ChartOpts | LocalChartOpts,
         opts?: pulumi.ComponentResourceOptions
     ) {
+        pulumi.log.warn("Chart is deprecated: helm/v2/Chart is deprecated by helm/v3/Chart and will be removed in a future release.")
```

Comment:
We should also add `@deprecated` JSDoc tag to the bottom of the class's doc comment ([like this](https://github.com/pulumi/pulumi-aws/blob/381d01b575bb9f5b449b8f4992ed3145bd0d5ffa/sdk/nodejs/ses/confgurationSet.ts#L7-L9)). Which now has nice [IDE support in VS Code](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-0.html#-deprecated--support).